core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -218,9 +179,9 @@ public void init() throws SchemaRegistryInitializationException {
     }
 
     try {
-      createZkNamespace();
-      masterElector = new ZookeeperMasterElector(zkUtils, myIdentity, this,
+      masterElector = new ZookeeperMasterElector(config, myIdentity, this,
                                                  isEligibleForMasterElector);
```

Comment:
may be we should have a configure method in the interface just so that the elector class can be made configurable.